docs/webdriver-vs-protractor.md

```diff
@@ -0,0 +1,28 @@
+Webdriver Syntax vs Protractor Syntax
+=====================================
+
+In vanilla WebDriver code, you might start your tests with 
```

Comment:
Let's change WebDriver to WebDriverJS and add a link to https://code.google.com/p/selenium/wiki/WebDriverJs